tests/find_cmd_tests.rs

```diff
@@ -587,3 +587,54 @@ fn find_newer_xy() {
         }
     }
 }
+
+#[test]
+#[serial(working_dir)]
+#[cfg(target_os = "linux")]
```

Comment:
```suggestion
#[cfg(unix)]
```

Reply:
Thanks. Modified in latest commit. This target is more suitable than `linux`. : )